### PR TITLE
Add Dusko Valve proposal

### PR DESCRIPTION
**Pull request by mikeosborne**

### CODE REVIEW

File: content/grants/dusko-valve.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Artists such as Haleek Maul have created custom collections of Urbit sigils which auctioned off both the artwork and corresponding planet or star together. This was a great example of artists selling artwork that immediately grants access to the Urbit community. Making a tool to do this at scale would further Urbit’s goal of becoming the primary social space for web3, as well as promote Urbit as an attractive, alternative platform to smaller NFT artists who don’t want to operate and tend to Discords, but are still interested and excited by the notion of communicating more directly with their supporters.
+
+## Implementation
+
+Money from this grant would go to creating a tool for NFT artists selling artwork to provide their community immediate access to Urbit by redeeming a planet.
+This would initially be an integration with Lit Protocol which queries an artist’s NFT contract to provide access to a modified planettine-based react app for distributing planets.
```
Comment (tacryt-socryp) [inline content/grants/dusko-valve.md]: Please update to the current light metamask-based auth implementation.

File: content/grants/dusko-valve.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Stages of Valve
+
+1. The Valve MVP will be a redeployment of Tlon’s planettine distribution tool behind a Lit Protocol authentication layer. Planet slugs will be maintained in an airtable to be available for redemption through planettine’s react app. Tlon will work with me to polish onboarding copy for the frontend which will clearly explain how an NFT holder redeems, then boots up, their planet. Tlon will also host all the planets of the Dusko community, starting with the core team.
```
Comment (tacryt-socryp) [inline content/grants/dusko-valve.md]: *planentine. Also, given you're no longer using Lit protocol, please specify that you'll be using a lighter metamask-based auth.

File: content/grants/dusko-valve.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## Milestones
+
+### Milestone 1 - MVP
+
+Payment: $4,000 in WSTR
+
+Completion: 30 Apr 2022
+
+MVP: a web page gated using Lit Protocol which users are granted access to once they mint a corresponding NFT.
```
Comment (tacryt-socryp) [inline content/grants/dusko-valve.md]: Should be updated to specify use of metamask authentication instead

**Review by tacryt-socryp — COMMENTED** [review]
Looks good, just one comment that needs to be reflected across a few places regarding implementation.

**Comment by thelifeandtimes** [comment]
@mikeosborne I don't have edit permissions on this PR, but we need to collapse milestones 2 and 3 into a single milestone, and then make the compensation for each milestone 1 star; for 2 stars total.

**Comment by mikeosborne** [comment]
Requested changes made. Give me a final review and I'll have Josh merge.

**Review by thelifeandtimes — APPROVED** [review]
Looks good to me.

File: content/grants/dusko-valve.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Milestone 2 - Airtable form and Tirrel Backend
+
+Payment: 1 star
+
+Completion: 30 Jun 2022
+
+Airtable form for Valve which other artists fill out to request redeemable planets and at least two artists lined up to transition their community over to Urbit through a Valve NFT release.
+
+Implementation of Tirrel backend and open-sourcing of Valve through release of repo and guide.
+500 planets distributed in total with at least communities transitioned over to Urbit.
```
Comment (jalehman) [inline content/grants/dusko-valve.md]: "...at least communities..."

I feel like something is missing there.

**Review by jalehman — APPROVED** [review]
Looks good pending the one comment.

File: content/grants/dusko-valve.md (hunk 4 above)
Comment (thelifeandtimes) [reply content/grants/dusko-valve.md]: Looks like it should be 'at least **_2_** communities'? Good catch, thanks.

Comment (jalehman) [inline content/grants/dusko-valve.md]:
```suggestion
500 planets distributed in total with at least two communities transitioned over to Urbit.
```

**Pull request merged**